grafana/risingwave-dev-dashboard.dashboard.py

```diff
@@ -699,6 +699,20 @@ def section_streaming(panels):
                 )
             ]
         ),
+        panels.timeseries_count(
+            "Kafka Consumer Lag Size",
```

Comment:
Please refine this metrics... @tabVersion 

- "Kafka Consumer Lag Size" is not that easy to understand. Please write a better description for it
- `high_watermark` and `latest_message_id` is too general. Please add a suffix for it e.g. `kafka_source_watermark`. 
- Since you put 2 kinds of metrics in one panel, the legend names must be differentiated. e.g.

```diff
  f"{metric('high_watermark')}",
- "source={{source_id}} partition={{partition}}"
+ "high watermark @ source={{source_id}} partition={{partition}}"
```